src/index.test.ts

```diff
@@ -176,6 +176,17 @@ describe('basic retry and timeout usage', () => {
     expect(res).toEqual(success(2));
   });
 
+  it('retries and resolves successful asynchronous functions with varying timeouts', async () => {
+    jest.spyOn(operations, 'successFn');
+    const start = performance.now();
+    const res = await go(operations.successFn, { attemptTimeoutMs: [5, 10, 15, 25], retries: 3 });
```

Comment:
I'd bump the numbers a bit to avoid flakiness issues due to timings. In my experience 50ms seems to be enough to avoid flakiness. (e.g. 50, 50, 100, 100)

Reply:
bumped up the minimum to 50ms